### ISSUE TITLE
Cannot select Rapid objects when wireframe is engaged

### ISSUE THREAD

**Issue by bradlewt** [facebook/Rapid#983]
### Description

Normally one can select a Rapid object, like a building, hit A and place it.  Currently it is not possible to select the proposed object when wireframe is engaged.  It is only possible to select when the building is considered "full".  This obscures the imagery it makes it hard to determine if the building is correct, or not.

### Version

2.0.3

### What browser are you seeing the problem on? What version are you running?

Firefox v114.0

### The OS you're using

win

### Steps to reproduce

1. open new rapid instance.
2. go to an area without buildings
3. if on full, select proposed building
4. deselect said building
5. hit "w" to wireframe the object
6. try to select
7. hit "w" again to fill the object and observe it is now, again, select-able.

https://github.com/facebook/Rapid/assets/24398638/dcecc1bb-60cd-42ed-8a11-5d4f6884f8c1

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit#background=Mapbox&datasets=msBuildings&disable_features=landuse&map=20.52/39.16531/22.88964

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/114.0

**Comment by bhousel** [comment]
related #719 
tl;dr - how we are drawing polygons isn't ideal.  The fill/mask is the interactive part, so when it goes away, no pointer events occur 😆

**Comment by Bonkles** [comment]
Fixed this by mimicking the synthetic 'hit target' code for our ways and using it ONLY during the wifreframe case for polys. i.e. we generate a hit target for the surrounding polygon (not any of the inners) in wireframe mode. The width of the hit target is VERY small, in keeping with the spirit of wireframe mode. 

Vid: 


https://github.com/facebook/Rapid/assets/1887955/cacf1519-cda2-4b16-b610-0452feac7c56
